packages/realm-server/tests/realm-server-test.ts

```diff
@@ -1855,6 +1855,15 @@ module('Realm Server', function (hooks) {
           'embedded html looks correct (CardDef template)',
         );
 
+        // TODO: why is card-api module + css not present in the boxel index table?
```

Comment:
In this test case we are fetching prerendered cards without ON filter which returns embedded template for `CardDef`. Additionally it should also return the css entry for `https://cardstack.com/base/card-api` module but I discovered this module is not indexed - it's not present in the boxel index table during this test execution. @habdelra any tips on how to make sure `https://cardstack.com/base/card-api` gets indexed in this test?

Reply:
I don’t know. I think you will need to run the server in the debugger to find out why we aren’t visiting this module

Reply:
Ok. I thought maybe there is a special reason why it's not indexed. Will debug more

Reply:
I discovered the test database where the `https://cardstack.com/base/card-api` module gets indexed into is different from the one the test realm is reading from when making requests to it. Sounds like a test setup issue, will look into that

Reply:
I confirm that when passing the same adapter to both base and test realm (so that they use the same db), all of the modules in the test realm are getting indexed OK. However I am wondering if this is really needed... isn't the test realm supposed to index everything into its own database even if the base realm index runs in a different db?   Perhaps there is another issue here

Reply:
right now a realm only indexes instances/modules that are in its own realm--it doesn't index instances/modules of external realms.

Reply:
I fixed this by changing the realm tests so that the base realm and test realm now run in the same server. Previously, they were running in separate servers, and this made testing css payloads where css comes from different realms impossible